specification/containerinstance/resource-manager/readme.md

```diff
@@ -0,0 +1,72 @@
+# ContainerInstance
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for ContainerInstance.
+
+
+
+---
+## Getting Started 
+To build the SDK for ContainerInstance, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the ContainerInstance API.
+
+``` yaml
+openapi-type: arm
+tag: package-2017-08-preview
+```
+
+
+### Tag: package-container-instance-2017-08-preview
```

Comment:
Blocking: this tag is incorrect and should be `package-2017-08-preview` in the three places used